database/src/main/java/com/firebase/ui/database/FirebaseRecyclerAdapter.java

```diff
@@ -110,9 +110,25 @@ public void onChanged(EventType type, int index, int oldIndex) {
             public void onCancelled(DatabaseError databaseError) {
                 FirebaseRecyclerAdapter.this.onCancelled(databaseError);
             }
+
+            @Override
+            public void onReady() {
+                FirebaseRecyclerAdapter.this.onDataChanged();
```

Comment:
Wouldn't we want to unify these names and pick either `onReady` or `onDataChanged`? Other than that, LGTM!